app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  def recent_comments
+    comments.limit(5).order(created_at: :desc)
+  end
+
+  def update_posts_counter
+    user.increment!(:Posts_Counter)
```

Comment:
- [ ] You do not have `:Posts_Counter` attribute in your user table. What you have is `PostsCounter`. Kindly fix this.

- [ ] From the requirement above we should have `author.increment!`, not `user.increment!`

  ```
  def update_posts_counter
    author.increment!(:PostsCounter)
  end
  ```

- **[OPTIONAL]** I recommend you add a callback for this method, so that it can be called when a post is saved.

  ```
  after_save :update_posts_counter
  
  ....
  
  private
  
  def update_posts_counter
    user.increment!(:Posts_Counter)
  end
  ```